server/config/toml.go

```diff
@@ -182,6 +182,11 @@ max-recv-msg-size = "{{ .GRPC.MaxRecvMsgSize }}"
 # The default value is math.MaxInt32.
 max-send-msg-size = "{{ .GRPC.MaxSendMsgSize }}"
 
+# LogQueries if enabled will print an info log containing the query request
```

Comment:
Can you add the following here to keep confix up to date: https://github.com/cosmos/cosmos-sdk/blob/main/tools/confix/data/v0.51-app.toml ?

Reply:
Didn't know this existed! Added here https://github.com/cosmos/cosmos-sdk/pull/19049/commits/8e569442a13855cf7da79559fd2046b75e91ec8a